docs/documentation/maya/simple_setup.md

```diff
@@ -1,221 +1,297 @@
+<figure>
+  <img src="images/setup_skin_4.png"> 
+  <figcaption>Figure 5: Slide Constraints weights paint.</figcaption>
+</figure>
 
-With this basic paint setup the AdnSkin deformer will already show a proper behavior, expected of the skin to our reference target mesh. However, the possible parameters and tweaks to display high fidelity dynamics can be seen [here](skin.md).
+With this basic paint setup the AdnSkin deformer will already show plausible results, expected of the skin to the reference target mesh. However, the possible parameters and tweaks to display high fidelity dynamics can be seen in the documentation for [AdnSkin](skin.md).
 
-## AdnMuscle simple setup
+## AdnMuscle Simple Setup
 
-To set up AdnMuscle we will start with a scene with the following elements:
+To create a basic scenario using the AdnMuscle deformer, start with a scene with the following elements:
 
  - An animated rig.
  - A simulated mesh.
 
-In this case we will be simulating a biceps in an animated full body rig. The AdnMuscle deformer will be applied to the mesh of the biceps.
+In this case the proposed example is to simulate a biceps in an animated full body rig. The AdnMuscle deformer will be applied to the mesh of the biceps.
 
 <figure>
   <img src="images/setup_muscle_0.png"> 
-  <figcaption>Figure: Basic set up for biceps simulations.</figcaption>
+  <figcaption>Figure 6: Basic setup for biceps simulations.</figcaption>
 </figure>
 
 ### 1. Creating the deformer
 
-To create the deformer we will select the two joints of the rig that form the "arm" and the "forearm" and then the mesh of the muscle. Then we will press the ![adnMuscle](images/adn_muscle.png) shelf button or go to *AdonisFX menu > Muscle*.
+To create the AdnMuscle deformer, select the two joints of the rig that form the "arm" and the "forearm" and then the mesh of the muscle. Then, press the ![AdnMuscle](images/adn_muscle.png) shelf button or go to AdonisFX Menu > *Muscle*.
 
 <figure>
   <img src="images/setup_muscle_1.png"> 
-  <figcaption>Figure: Basic set up for biceps simulations.</figcaption>
+  <figcaption>Figure 7: AdnMuscle deformer creation scenario.</figcaption>
 </figure>
 
-If we want to create the muscle deformer with some initial customization we can double click the shelf button or press the option box in the menu item. This will display a popup window that will allow us to do some initial customization, as well as creating the deformer with a custom name. Once everything is in order press the "Create" button and the deformer will get created.
+To create the AdnMuscle deformer with some initial customization, double click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the "Create" button and the deformer will get created.
```

Comment:
To keep consistency:
```suggestion
To create the AdnMuscle deformer with some initial customization, double-click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the *Create* button and the deformer will get created.
```

Reply:
Done.